src/main/java/cz/codecamp/logger/loggers/Logger.java

```diff
@@ -0,0 +1,120 @@
+package cz.codecamp.logger.loggers;
+import cz.codecamp.logger.LogLevelEnum;
+import cz.codecamp.logger.LoggerInterface;
+import sun.rmi.runtime.Log;
+
+import java.io.IOException;
+import java.text.SimpleDateFormat;
+import java.util.*;
+
+/**
+ * Created by dmachace on 16.10.2016.
+ */
+public class Logger implements LoggerInterface {
+
+    private static final Properties props = Properties.INSTANCE;

```

Comment:
Žádný z následující fieldů by neměl být `static`:
- mít jedny statické `props` zhoršuje, spíše znemožňuje, testování
- `fileHandler` je vytvářený v konstruktoru, je tedy vázaný na instanci, `static` proměnné jsou pro všechny instance stejné
- `date` nedává asi vůbec smysl mít jako field
- `dateFormat` je navázaný na props - platí tam to stejné, mít ho jako `static` zhoršuje, znemožňuje, testování